src/arrays.js

```diff
@@ -1,61 +1,133 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length){
+  return array[index - array.length];
+  } else{
+  return array[index];
+  }
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  const array2 = array.concat(element);
+  return array2;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  let strings = [];
+  for (let i = 0; i < numbers.length; i += 1){
+    strings.push(numbers[i].toString());
+  }
+  return strings;
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  let uppercaseArray = [];
+  for (let i = 0; i < strings.length; i += 1){
```

Comment:
Another great use of a for loop, but try and rewrite this using one of the iterator functions we used on Wednesday